testing/integration-tests/agent-internals/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id("elastic-otel.test-with-agent-conventions")
+}
+
+tasks.test {
+  jvmArgs(
+    //"-Dotel.javaagent.debug=true"
```

Comment:
[minor] looks like leftover from debugging session, maybe we could have a way to consistently enable debug logging when running tests, or simply rely on the value of `OTEL_JAVAAGENT_DEBUG=true` which is also automatically enabled when debugging smoke tests.

Reply:
> enable debug logging when running tests

We could do this via the `test-with-agent-conventions` if you want to. I just felt like having debug always on on everything is a bit noisy.

I intentionally left it there as a comment to allow easy enabling of the debug logs.

> simply rely on the value of OTEL_JAVAAGENT_DEBUG=true which is also automatically enabled when debugging smoke tests.

What do you mean with `rely on the value of OTEL_JAVAAGENT_DEBUG=true` ? This is not a smoke-test, therefore the logic implemented there doesn't have any effect in these tests.